Workbooks/CiscoMeraki.json

```diff
@@ -0,0 +1,466 @@
+        "comparison": "isEqualTo",
+        "value": "ConfigurationChanges"
+      },
+      "customWidth": "33",
+      "name": "query - 8"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "MerakiConfigurationChanges\r\n| summarize PageCount=count() by page\r\n| sort by PageCount desc",
+        "size": 0,
+        "title": "Top Events - By Page",
+        "timeContext": {
+          "durationMs": 2592000000
+        },
+        "timeContextFromParameter": "TimeRange",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "visualization": "table",
+        "gridSettings": {
+          "formatters": [
+            {
+              "columnMatch": "OperationCount",
+              "formatter": 3,
+              "formatOptions": {
+                "showIcon": true
+              }
+            }
+          ]
+        }
+      },
+      "conditionalVisibility": {
+        "parameterName": "Tab",
+        "comparison": "isEqualTo",
+        "value": "ConfigurationChanges"
+      },
+      "customWidth": "33",
+      "name": "query - 7 - Copy"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "MerakiConfigurationChanges\r\n| summarize LabelCount=count() by label\r\n| sort by LabelCount desc",
+        "size": 0,
+        "title": "Top Events - By Label",
+        "timeContext": {
+          "durationMs": 2592000000
+        },
+        "timeContextFromParameter": "TimeRange",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "visualization": "table",
+        "gridSettings": {
+          "formatters": [
+            {
+              "columnMatch": "OperationCount",
+              "formatter": 3,
+              "formatOptions": {
+                "showIcon": true
+              }
+            }
+          ]
+        }
+      },
+      "conditionalVisibility": {
+        "parameterName": "Tab",
+        "comparison": "isEqualTo",
+        "value": "ConfigurationChanges"
+      },
+      "customWidth": "33",
+      "name": "query - 7"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "MerakiConfigurationChanges\r\n| sort by Timestamp desc",
+        "size": 0,
+        "title": "Raw Audit Events",
+        "timeContext": {
+          "durationMs": 2592000000
+        },
+        "timeContextFromParameter": "TimeRange",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "gridSettings": {
+          "filter": true
+        }
+      },
+      "conditionalVisibility": {
+        "parameterName": "Tab",
+        "comparison": "isEqualTo",
+        "value": "ConfigurationChanges"
+      },
+      "name": "query - 35"
+    }
+  ],
+  "fallbackResourceIds": [
+    ""
+  ],
+  "fromTemplateId": "sentinel-UserWorkbook",
```

Comment:
Can you please change the value of fromTemplateId in workbook to "sentinel-CiscoMeraki" as it should be sentinel-workbookname. Please refer the documentation here https://github.com/Azure/Azure-Sentinel/blob/45310fe6be3cb30cbd8f073443a0c2f9144115fa/Workbooks/README.md

Reply:
Added and committed

Reply:
> Can you please change the value of fromTemplateId in workbook to "sentinel-CiscoMeraki" as it should be sentinel-workbookname. Please refer the documentation here https://github.com/Azure/Azure-Sentinel/blob/45310fe6be3cb30cbd8f073443a0c2f9144115fa/Workbooks/README.md

@richlilly2004 Please update the value of  fromTemplateId in workbook to "sentinel-CiscoMeraki", you have by mistakenly updated them in playbooks.

Thanks,
Aditi Maheshwari

Reply:
@richlilly2004 Please update the value of fromTemplateId in workbook to "sentinel-CiscoMeraki", you have by mistakenly updated them in playbooks.

Thanks,
Aditi Maheshwari

Reply:
@richlilly2004  Every thing related to workbook looks good , only fromTemplateId  value should be updated to sentinel-CiscoMeraki for approval.

could you please update the value of fromTemplateId in workbook?

Thanks,
Aditi Maheshwari

T